src/index.js

```diff
@@ -78,7 +78,7 @@ module.exports = {
       }
     },
     queue(name, action, params, options) {
-      return this.$resolve(name).add(action, { params, meta: this.currentContext.meta }, options)
+      return this.$resolve(name).add(action, { params, meta: {} }, options)
```

Comment:
This is not possible because job will not have the current meta (Let say an user profile coming from the web gateway)
We can do something like that : 
```js
    queue(ctx, name, action, params, options) {
      return this.$resolve(name).add(action, { params, meta: ctx.meta }, options)
      return this.$resolve(name).add(action, { params, meta: {} }, options)
    }
```
And update the README about that (I will update to a major version because of this breaking change)